pkg/infrastructure/services/errors.go

```diff
@@ -25,6 +25,7 @@ const (
 	reconciliationIntervalAfterMessagingError            = time.Second * 30
 	reconciliationIntervalMonitoringEndpointNotAvailable = time.Second * 10
 	reconciliationIntervalAfterDashboardsError           = time.Second * 30
+	reconciliationIntervalAfterFetchService              = time.Minute * 3
```

Comment:
isn't 3 minutes too much? is there any service that takes it to start/be ready?

Reply:
Normally this is related to a malformed URL or a URL that is not being resolved, not the service itself. 99.9999% of the time in a local env, that's why.